bot/exts/valentines/valentine_zodiac.py

```diff
@@ -29,13 +37,127 @@ def load_json() -> dict:
+            if date >= 1 and date <= 19:
+                zodiac = "Aries"
+            elif date >= 20 and date <= 30:
+                zodiac = "Taurus"
+            else:
+                zodiac = None
+        elif month == "May":
+            if date >= 1 and date <= 20:
+                zodiac = "Taurus"
+            elif date >= 21 and date <= 31:
+                zodiac = "Gemini"
+            else:
+                zodiac = None
+        elif month == "June" or month == "Jun":
+            if date >= 1 and date <= 20:
+                zodiac = "Gemini"
+            elif date >= 21 and date <= 30:
+                zodiac = "Cancer"
+            else:
+                zodiac = None
+        elif month == "July" or month == "Jul":
+            if date >= 1 and date <= 22:
+                zodiac = "Cancer"
+            elif date >= 23 and date <= 31:
+                zodiac = "Leo"
+            else:
+                zodiac = None
+        elif month == "August" or month == "Aug":
+            if date >= 1 and date <= 22:
+                zodiac = "Leo"
+            elif date >= 23 and date <= 31:
+                zodiac = "Virgo"
+            else:
+                zodiac = None
+        elif month == "September" or month == "Sept":
+            if date >= 1 and date <= 22:
+                zodiac = "Virgo"
+            elif date >= 23 and date <= 30:
+                zodiac = "Libra"
+            else:
+                zodiac = None
+        elif month == "October" or month == "Oct":
+            if date >= 1 and date <= 22:
+                zodiac = "Libra"
+            elif date >= 23 and date <= 31:
+                zodiac = "Scorpio"
+            else:
+                zodiac = None
+        elif month == "November" or month == "Nov":
+            if date >= 1 and date <= 21:
+                zodiac = "Scorpio"
+            elif date >= 22 and date <= 30:
+                zodiac = "Sagittarius"
+            else:
+                zodiac = None
+        elif month == "December" or month == "Dec":
+            if date >= 1 and date <= 21:
+                zodiac = "Sagittarius"
+            elif date >= 22 and date <= 31:
+                zodiac = "Capricorn"
+            else:
+                zodiac = None
+        else:
+            zodiac = None
+            log.info("Wrong Zodiac date or month provided")
+        return zodiac
+        log.info("Zodiac name sent")
+
+    @commands.group(name="partnerzodiac", invoke_without_command=True)
+    async def partner_zodiac(self, ctx: commands.Context, zodiac_sign: str) -> None:
         """Provides a counter compatible zodiac sign to the given user's zodiac sign."""
         try:
             compatible_zodiac = random.choice(self.zodiacs[zodiac_sign.lower()])
         except KeyError:
-            return await ctx.send(zodiac_sign.capitalize() + " zodiac sign does not exist.")
+            return await ctx.send(f"`{zodiac_sign.capitalize()}` zodiac sign does not exist.")
```

Comment:
the function annotation says the return Type is None, so we have to refactor our code to match that
`await ctx.send(f"`{zodiac_sign.capitalize()}` zodiac sign does not exist.")`
`return`